src/Akka.HealthCheck.IHealthCheckExample/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Akka.Cluster.Hosting;
+using Akka.HealthCheck.Cluster;
+using Akka.HealthCheck.IHealthCheckExample;
+using Akka.HealthCheck.IHealthCheckExample.HealthChecks;
+using Akka.HealthCheck.Liveness;
+using Akka.HealthCheck.Persistence;
+using Akka.HealthCheck.Readiness;
+using Akka.Hosting;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+builder.Services.AddAkka("actor-system", (configurationBuilder, serviceProvider) =>
+{
+    configurationBuilder
+    .WithClustering()
+    .WithActors((actorSystem, registry) =>
+    {
+        // We need to start the Probe actors which we need in our HealthChecks
+        actorSystem.CreateAndRegisterHealthProbe<DefaultLivenessProbe>(registry, new DefaultLivenessProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<ClusterLivenessProbe>(registry, new ClusterLivenessProbeProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<AkkaPersistenceLivenessProbe>(registry, new AkkaPersistenceLivenessProbeProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<DefaultReadinessProbe>(registry, new DefaultReadinessProvider(actorSystem));
+        actorSystem.CreateAndRegisterHealthProbe<ClusterReadinessProbe>(registry, new ClusterReadinessProbeProvider(actorSystem));
+    });
+});
+
+// Add health checks - look here for documentation: https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-6.0
+builder.Services.AddHealthChecks()
+    .AddCheck<AkkaLivenessActorHealthCheck<ClusterLivenessProbe>>("akka-liveness-cluster", HealthStatus.Unhealthy, new[] { "liveness" })
```

Comment:
I think you would not want to bind these automatically but just want to leave that to the consumer of the library so that they have the full flexibility over where to use which healthchecks.